eng/scripts/smoketest/Save-Package-Properties.ps1

```diff
@@ -0,0 +1,122 @@
+<#
```

Comment:
Why are we duplicating this script? What is different from the common one?

Reply:
This script only picks out the track 2 packages, the original one grabs aeverything

Reply:
I'm not sure this is true; I ran this locally and am seeing the T1 packages from the `eventhub` service directory in the output.

Reply:
Ignoring the track2 vs not logic we should definitely not fully duplicate the script to add that level of filtering. 

Looks like it should filter down to new sdk's https://github.com/Azure/azure-sdk-for-net/pull/25673/files#diff-0b196dd028dd0152730b5a60328c42945f7371f0301f8ff5a4e1b01f90edcf97R98.